cmd/opm/registry/serve.go

```diff
@@ -106,7 +109,27 @@ func serveFunc(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		logger.Fatalf("failed to listen: %s", err)
 	}
+
+	timeout, err := cmd.Flags().GetString("timeout-seconds")
+	if err != nil {
+		return err
+	}
+
 	s := grpc.NewServer()
+	logger.Printf("Keeping server open for %s seconds", timeout)
+	if timeout != "infinite" {
+		timeoutSeconds, err := strconv.ParseInt(timeout, 10, 16)
```

Comment:
anything stopping people from inserting negative number?

Reply:
We should use `ParseUint` here to avoid negative numbers.

Reply:
Fixed.